### PR TITLE
Update import documentation for Emacs

### PR DESCRIPTION
**Pull request by kostafey**
Related to issue https://github.com/scalameta/metals/issues/849#issuecomment-558308394

### CODE REVIEW

File: metals-docs/src/main/scala/docs/TextEditorModifier.scala [hunk 1]
```diff
@@ -12,6 +12,11 @@ class TextEditorModifier extends StringModifier {
       input: Input,
       reporter: Reporter
   ): String = {
+    val autoImportNote = if editor == "emacs" {
```
Comment (olafurpg) [inline metals-docs/src/main/scala/docs/TextEditorModifier.scala]:
```suggestion
    val autoImportNote = if (editor == "emacs") {
```

**Comment by tgodzik** [comment]
@kostafey Thanks for following up with this - it took the liberty of fixing the compilation issue :sweat_smile: .

**Comment by tgodzik** [comment]
It seems docusaurus step is failing.

@olafurpg any idea what's going on? I can't reproduce locally.
```
$ docusaurus-build
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module './src/data'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/runner/work/metals/metals/website/node_modules/core-js-compat/helpers.js:3:14)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Module._compile (/home/runner/work/metals/metals/website/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.newLoader [as .js] (/home/runner/work/metals/metals/website/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:653:32)
error Command failed with exit code 1.
```

**Comment by ckipp01** [comment]
> It seems docusaurus step is failing.

I played around with this to see if I could replicate it, but couldn't. Are we sure this isn't just by chance? Are we able to run the check again?

**Comment by olafurpg** [comment]
@kostafey can you please force-push again to your PR? I'm unable to restart the CI for some reason

**Comment by tgodzik** [comment]
I might have broken github a bit, will reopen a new PR :O

**Pull request closed without merging**